package.json

```diff
@@ -111,6 +111,11 @@
         "required": false,
         "default": "default"
       },
+      "parametersToHeaders": {
+        "description": "If true, this will create headers on the incoming messages for each channel parameter. Currently this only works with messages originating from Solace, which have the solace_destination header set.",
```

Comment:
what would have to be done to extend support for not Solace only?
also please update Readme

Reply:
I need to do some research to see whether other protocols make their topics available somehow. With Solace, you can get the topic from the solace_destination header. I don't yet know if AMQP or MQTT do something similar. This feature would be meaningless with Kafka because Kafka topics are static. I'll update the README after I learn some more about this.

Reply:
Ideally this same feature should work for Kafka & Rabbit (AMQP 0.9.1) binders as well since those binders are supported by this template....

1. For Rabbit it looks like the channel represents either a routing key or queue. Queues are static so this probably only needs to be implemented for routing keys (which is the default).
2. As Michael mentioned, Kafka topics are static so it's not likely that a "variable" would be in the topic name. Until someone comes across a use case with Kafka where this would be useful I think we can skip implementing this for Kafka.

Reply:
It now works with Rabbit!

Reply:
Please update the description as it doesn't match the one from readme

Reply:
@damaru-inc ☝🏼

Reply:
Done!